doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -111,6 +111,8 @@ The procedure to install CORTX on OVA is mentioned below.
                
     salt '*' cmd.run "firewall-cmd --zone=public-data-zone --add-port=80/tcp --permanent"
     
+    salt '*' cmd.run "firewall-cmd --zone=public-data-zone --add-port=28100/tcp --permanent"
```

Comment:
@mukul-seagate11  - I feel this should be handled during Provisioning to avoid this manual step. Tagging @ypise for guidance and recommendation.

Reply:
@mukul-seagate11 , 
I would prefer executing only single command to apply a salt state to ensure all firewall rules are applied correctly:
`salt "*" state.apply components.system.firewall`

It would need to be verified on a system though.

Reply:
OK sounds good so can this be taken forward to get fixed in next version/release to avoid executing manually as I see below commands updated in https://github.com/Seagate/cortx/blob/main/doc/CORTX_on_Open_Virtual_Appliance.rst

![image](https://user-images.githubusercontent.com/65269273/110295327-ff8d0580-8016-11eb-81d8-8c9afc3dcc5e.png)

![image](https://user-images.githubusercontent.com/65269273/110295467-264b3c00-8017-11eb-9f10-94dcccf16133.png)

above updated in https://github.com/Seagate/cortx/blob/main/doc/Preboarding_and_Onboarding.rst